web/auth.go

```diff
@@ -413,25 +413,68 @@ type tokenResponse struct {
 	Expiry       string `json:"expiry"`
 }
 
+func getBearerToken(req *http.Request) string {
+	authHeader := req.Header.Get("Authorization")
+	if authHeader == "" {
+		return ""
+	}
+
+	parts := strings.Split(authHeader, " ")
+	if len(parts) == 2 && parts[0] == "Bearer" {
+		return parts[1]
+	}
+
+	return ""
+}
+
 func (h *tokenHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx, cancel := context.WithTimeout(r.Context(), 10*time.Second)
 	defer cancel()
 
 	_, span := trace.StartSpan(ctx, r.URL.Path)
 	defer span.End()
 
+	current := getBearerToken(r)
+	failedToGetCookies := false
+
 	token, err := r.Cookie("viam.auth.token")
-	if HandleError(w, err, h.logger, "getting token cookie") {
-		return
+	if err != nil || token.Value == "" {
+		failedToGetCookies = true
 	}
 
 	refresh, err := r.Cookie("viam.auth.refresh")
-	if HandleError(w, err, h.logger, "getting refresh cookie") {
-		return
+	if err != nil || token.Value == "" {
+		failedToGetCookies = true
 	}
 
 	expiry, err := r.Cookie("viam.auth.expiry")
-	if HandleError(w, err, h.logger, "getting expiry cookie") {
+	if err != nil || token.Value == "" {
+		failedToGetCookies = true
+	}
```

Comment:
are these supposed to be checking `token.Value` from `r.Cookie("viam.auth.token")` or is it supposed to be checking `current`?

Reply:
if its the former, can we get rid of it in the second two checks since it will always be true if its true the first time?

Reply:
Nope, just some copy paste errors, will resolve them.